setup.py

```diff
@@ -19,50 +19,54 @@
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 #
 #
-from setuptools import setup, find_packages
-
+from setuptools import find_packages, setup
 
 with open("README.md", "r") as f:
     long_description = f.read()
 
+# fmt: off
 setup(
     name='bluetooth-mesh',
-    version='0.1.29',
-    author='Michał Lowas-Rzechonek',
+    version='0.2.0',
     author_email='michal.lowas-rzechonek@silvair.com',
     description=(
-        'Bluetooth Mesh for Python'
+        'Bluetooth mesh for Python'
     ),
     long_description=long_description,
     long_description_content_type="text/markdown",
     url='http://github.com/silvairgit/python-bluetooth-mesh',
     packages=find_packages(exclude=('test*', )),
-    python_requires='>=3.5.0',
+    python_requires='>=3.6.0',
+    setup_requires=[
+        'pytest-runner>=4.2',
+    ],
     install_requires=[
         'bitstring>=3.1.5',
-        'construct>=2.9.45,<2.10.0',
+        'construct==2.9.45',
         'cryptography>=2.3.1',
+        'crc==0.3.0',
+        'dbus-next==0.1.3',
         'ecdsa==0.15',
-        'crc==0.3.0'
     ],
-    extras_require={
-        'demo': [
-            'docopt>=0.6.2',
-            'marshmallow>=3.0.0rc3',
-            'pillow>=5.4.1',
-            'pygobject>=3.30.4',
-            'pydbus>=0.6.0',
-            'prompt-toolkit>=2.0.8',
-        ],
-    },
     tests_require=[
+        'asynctest>=0.12.3',
         'pytest>4.1.0',
-        'pytest-runner>=4.2',
+        'pytest-asyncio>=0.10.0',
     ],
+    extras_require=dict(
+        doc=[
+            'sphinx>=2.1.2',
+            'sphinx-autodoc-typehints>=1.7.0',
+        ],
+        tools=[
+            'prompt-toolkit==2.0.10',
+            'platforms-clients>=4.39.4',
```

Comment:
As mentioned above, I'd like to remove this dependency completely.

Reply:
Mixed feeling, but see the last commit.

Reply:
I like this idea. The hidden `platforms-clients` import seems dirty, but this makes sense in the general sense.
Will you also implement the local storage network DB in this PR or in the next?

Reply:
I'm going to replace the "dirty" import with https://pypi.org/project/pluggy/-based hook.

Reply:
Are you going to do that in this PR or the next?

Reply:
I'd like to do it within this PR.